SU-HSDO/suhumsci
# ARCH-50 Add paragraphs between others

# READY FOR REVIEW

# Summary
- Adds the ability to add a paragraph between others.
- This is different than the built in ability to "Add Above" because the built in functionality requires a modal paragraph selection. And the modal selection breaks on material theme.
- Ignore remaining codeclimate issues. These will be resolved later when i adjust codeclimate config.

# Needed By (Date)
- Whenever

# Urgency
- Low

# Steps to Test

1. checkout this branch
1. sync site `blt drupal:sync --site=archaeology --sync-files`
1. edit the home page
1. hover over existing paragraphs. a "+" icon should show on the right
1. click that icon. and choose a paragraph from the options
1. validate the correct paragraph is added in the correct location.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
